cmd/api/src/queries/graph_test.go

```diff
@@ -101,12 +101,15 @@ func TestGraphQuery_RawCypherSearch(t *testing.T) {
 	// This will be set to a default of 30 min, with a reduction factor of 3, so we should have a 10 min config timeout
```

Comment:
This comment should be updated to reflect the new default of 15 min as mentioned in the comment below.